### PR TITLE
refactor: :memo: aggregate text on technical requirements

### LINKED ISSUE #92: Aggregate all technical requirement text
Requirements from a developer perspective

### PR DESCRIPTION
**Pull request by signekb**
## Description

- This PR attempts to aggregate sections on technical requirements (from a developer perspective) from multiple pages to one page.

## Related Issues

Closes #92 

See also Issues #...

## Reviewer Focus

This PR only needs a quick review.

## Checklist

For general documentation:

- [X] Spell-check
- [X] Did the page(s) preview correctly on your machine without breaking
- [ ] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

File: software/technical-requirements.qmd [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+|-------------------------------|-----------------------------------------|
+| Follow standard styling (e.g., [PIP3](https://pypi.org/project/pip/) and [black](https://github.com/psf/black) formatting) | Coding style is vital to readability, so we follow best practices                                                                       |
+| Use [CalVer](https://calver.org/) versioning                                                                               | This style is simple and easily shows the age of the software                                                                           |
+| Incorporate test-driven and documentation-driven development styles                                                        | Each has pros and cons, we use what works best. See the [Contributing Guidelines](https://seedcase-project.org/community/contributing). |
+
+: Conventions that we follow for the Seedcase framework.
```
Comment (signekb) [inline software/technical-requirements.qmd:108]: This section could be moved to the Style documentation?

File: software/constraints.qmd [hunk 2]
```diff
@@ -5,28 +5,14 @@ order: 2
 
 {{< include /includes/_wip.qmd >}}
 
-This document lists technical, political/organizational, and
-convention-based constraints that we must consider when developing
-software for the Seedcase Project.
+This document lists organizational and  political/legal constraints
```
Comment (lwjohnst86) [inline software/constraints.qmd:8]:
```suggestion
This document lists organizational and political/legal constraints 
```

File: software/solution-strategy.qmd [hunk 3]
```diff
@@ -7,36 +7,10 @@ order: 4
 
 This document describes what overall strategy we have decided on to
 solve our problem, as outlined in the [Introduction
-section](introduction.qmd). Specifically, what main technologies and
-what architectural designs we will use. Any solution we decide must
-adhere to the [Principles](principles.qmd)
+section](introduction.qmd). Any solution we decide must
+adhere to the [Guiding principles](introduction.qmd#guiding-principles)
```
Comment (lwjohnst86) [inline software/solution-strategy.qmd:11]:
```suggestion
adhere to the [Guiding principles](principles.qmd)
```

File: software/technical-requirements.qmd (hunk 1 above)
Comment (lwjohnst86) [reply software/technical-requirements.qmd:108]: Good idea. Once it is setup more, we can move it over ☺️

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Nice 🥳 only small suggestions.

File: software/solution-strategy.qmd (hunk 3 above)
Comment (signekb) [reply software/solution-strategy.qmd:11]: Ah! I forgot to change this. Nice catch!

File: software/technical-requirements.qmd (hunk 1 above)
Comment (signekb) [reply software/technical-requirements.qmd:108]: Alright. Will leave it here for now then

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**